deals/module/deals.go

```diff
@@ -31,7 +31,7 @@ import (
 const (
 	chanWriteTimeout = time.Second
 
-	defaultDealStartOffset = 72 * 60 * 60 / util.EpochDurationSeconds // 72hs
+	defaultDealStartOffset = 48 * 60 * 60 / util.EpochDurationSeconds // 48hs
```

Comment:
Still, a question mark which is the right value for this. 
Eventually, we can move it to the Storage Config and let people have the change to play with it, but most prob they won't know the consequences yet.